<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnConfigDialog</class>
 <widget class="QDialog" name="ColumnConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>カラム設定</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing"><number>4</number></property>
   <property name="leftMargin"><number>8</number></property>
   <property name="topMargin"><number>8</number></property>
   <property name="rightMargin"><number>8</number></property>
   <property name="bottomMargin"><number>8</number></property>
   <item>
    <widget class="QWidget" name="gridContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomArea">
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0">
        <size><width>0</width><height>0</height></size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightControls">
       <item>
        <layout class="QHBoxLayout" name="navRow">
         <item>
          <widget class="QPushButton" name="btnFilter">
           <property name="text"><string>全て</string></property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPrev">
           <property name="text"><string>&lt;</string></property>
           <property name="maximumSize">
            <size><width>40</width><height>16777215</height></size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPage">
           <property name="text"><string>1/1</string></property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNext">
           <property name="text"><string>&gt;</string></property>
           <property name="maximumSize">
            <size><width>40</width><height>16777215</height></size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColumnConfigDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColumnConfigDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
